content/20.treatments.md

```diff
@@ -517,6 +516,43 @@ Supplementation of vitamin D and maintaining a healthy diet for optimum vitamin
+Some meta-analyses that have assessed the efficacy of probiotics in viral respiratory infections have reported moderate reductions in the incidence and duration of infection [@doi:10.1002/14651858.CD006895.pub3; @doi:10.1017/S0007114514000075]. 
+Indeed, randomized controlled trials have shown that administering _Bacillus subtilis_ and _Enterococcus faecalis_ [@doi:10.1007/s00134-016-4303-x], _Lactobacillus rhamnosus GG_ [@doi:10.1164/rccm.200912-1853OC], or _Lactobacillus casei_ and _Bifidobacterium breve_ [@doi:10.1186/s13054-018-2167-x] via the nasogastric tube to ventilated patients reduced the occurrence of ventilator-associated pneumonia in comparison to the respective control groups in studies of viral infections and sepsis. 
+These findings are supported by a recently published meta-analysis [@doi:10.4187/respcare.07097].
+There is a significant risk of ventilator-associated bacterial pneumonia in COVID-19 patients [@doi:10.6084/m9.figshare.12340496], but it can be challenging for clinicians to diagnose this infection due to the fact that severe COVID-19 infection presents with the symptoms of pneumonia [@doi:10.1186/s13054-020-03013-2].
+Therefore, an effective prophylactic therapy for ventilator-associated pneumonia in severe COVID-19 patients would be of significant therapeutic value.
+
+Probiotics are generally synonymous with the treatment of gastrointestinal issues due to their supposed anti-inflammatory and immunomodulatory effects [@doi:10/d2qp]. 
+Notably, gastrointestinal symptoms commonly occur in COVID-19 patients [@doi:10.1053/j.gastro.2020.03.020], and the ACE2 receptor is highly expressed in enterocytes of the ileum and colon, suggesting that these organs may be a potential route of infection [@doi:10.1101/2020.01.30.927806; @doi:10.1038/cr.2016.152]. 
+Indeed, SARS-CoV-2 viral RNA has been detected in human feces [@doi:10/ggq8zp] and fecal-oral transmission of the virus has not yet been ruled out [@doi:10/ggpx7s]. 
+Rectal swabs of some SARS-CoV-2 positive pediatric patients persistently tested positive for several days despite negative nasopharyngeal tests, indicating the potential for fecal viral shedding [@doi:10.1038/s41591-020-0817-4].
+However, there is conflicting evidence for the therapeutic value of various probiotics against the incidence or severity of gastrointestinal symptoms in viral or bacterial infections such as gastroenteritis [@doi:10.1007/s00203-017-1400-3; @doi:10.1056/NEJMoa1802597]. 
+Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19, a hypothesis that several clinical trials are now preparing to investigate [@clinicaltrials:NCT04420676; @clinicaltrials:NCT04390477; @clinicaltrials:NCT04366180]. 
```

Comment:
So looking at NCT04390477, it seems like their analysis is focused on the effect of probiotics on mortality/ time to discharge/ viral clearance. NCT04366180 does seem to evaluate GI symptoms but the primary outcome is SARS-CoV-2 infection. I think it might make sense to rephrase this sentence to be a little less specific about what outcome is being measured.

Reply:
```suggestion
Nevertheless, it has been proposed that the administration of probiotics to COVID-19 patients and healthcare workers may prevent or ameliorate the gastrointestinal symptoms of COVID-19, a hypothesis that several clinical trials are now preparing to investigate [@clinicaltrials:NCT04420676; @clinicaltrials:NCT04366180]. Whereas other studies are investigating whether probiotics may affect patient outcomes following SARS-CoV-2 infection [@clinicaltrials:NCT04390477]
```

Reply:
Does this split that up properly? @rando2

Reply:
Looks good to me!